Makefile

```diff
@@ -75,7 +75,7 @@ start: _prepare  ## Start a local Flyte sandbox
 	kubectl wait --for=condition=available deployment/flyteadmin -n flyte --timeout=10m
 
 	$(call LOG,Registering examples from commit: latest)
-	REGISTRY=ghcr.io/flyteorg VERSION=latest $(MAKE) fast_register
+	$(call RUN_IN_SANDBOX,bash -c "REGISTRY=ghcr.io/flyteorg VERSION=latest make -C cookbook/$(EXAMPLES_MODULE) fast_register")
```

Comment:
Should we call this after flyteadmin is up?

Reply:
yeah... I wait for it just two lines above..

Reply:
Oh right. Sorry I missed that completely!